package.json

```diff
@@ -90,7 +90,7 @@
     "@angular/platform-browser-dynamic": "^15.2.9",
     "@angular/router": "^15.2.9",
     "@edium/fsm": "^2.1.2",
-    "@hmcts/ccd-case-ui-toolkit": "7.0.10",
+    "@hmcts/ccd-case-ui-toolkit": "7.0.8-compress-ls",
```

Comment:
@OgunyemiO can you revert the TL version if you have not done any toolkit changes